assets-src/styles/sass/30-base/_tables.scss

```diff
@@ -34,12 +34,9 @@ table caption {
 }
 
 th {
+    background-color: $deep-blue;
```

Comment:
Discussing with @vivienlacourba, only the `<th>`s in a `<thead>` or `<tfoot>` should have that blue background.

Reply:
Have pushed a further changed for those requirements.